CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+0.8.1 / 2024-06-07
+===================
+## Kover Gradle Plugin
+
+### Features
+* [`#600`](https://github.com/Kotlin/kotlinx-kover/issues/600) Added DSL to copy one report variant
+* Added DSL to copy one report variant
```

Comment:
This is a duplication of the previous point. Perhaps you wanted to mention DSL enhancement?